FamilyTreeTesting/Scripts/FamilyTreeJs.js

```diff
@@ -777,7 +777,7 @@ function decreaseZoom() {
 function freeDraw() {
     myDiagram.toolManager.panningTool.isEnabled = false;
     // create drawing tool for myDiagram, defined in FreehandDrawingTool.js
-    tool = new FreehandDrawingTool();
+    var tool = new FreehandDrawingTool();
```

Comment:
this is wrong, you should also set global.tool to null after removing it from toolManager. This should happened in line 798.